modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -38,19 +39,94 @@ $ oc annotate Network.operator.openshift.io cluster \
   'networkoperator.openshift.io/network-migration'=""
 ----
 
-. To change the default CNI network provider, enter the following command:
+. To stop all of the machine configuration pools managed by the Machine Config Operator (MCO), enter the following commands:
++
+[source,terminal]
+----
+$ oc patch MachineConfigPool master --type='merge' --patch \
+  '{ "spec": { "paused": true } }'
+----
++
+[source,terminal]
+----
+$ oc patch MachineConfigPool worker --type='merge' --patch \
+  '{ "spec":{ "paused" :true } }'
+----
+
+. Configure the OVN-Kubernetes cluster network provider with one of the following commands:
+
+.. To specify the network provider without changing the cluster network IP address block, enter the following command:
 +
 [source,terminal]
 ----
 $ oc patch Network.config.openshift.io cluster \
   --type='merge' --patch '{ "spec": { "networkType": "OVNKubernetes" } }'
 ----
 
-. To confirm the migration disabled the OpenShift SDN default CNI network provider and removed all OpenShift SDN Pods, enter the following command. It might take several moments for all the OpenShift SDN Pods to stop.
+.. To specify a different cluster network IP address block, enter the following command:
++
+[source,terminal]
+----
+$ oc patch Network.config.openshift.io cluster \
+  --type='merge' --patch '{
+    "spec": {
+      "clusterNetwork": [
+        "cidr": "<cidr>",
+        "hostPrefix": "<prefix>"
+      ]
+      "networkType": "OVNKubernetes"
+    } 
+  }'
+----
++
+where `cidr` is a CIDR block and `prefix` is the slice of the CIDR block apportioned to each node in your cluster.
++
+IMPORTANT: You cannot change the service network address block during the migration.
+
+. Optional: You can customize the following settings for OVN-Kubernetes to meet your network infrastructure requirements:
++
+--
+* Maximum transmission unit (MTU)
+* Geneve (Generic Network Virtualization Encapsulation) overlay network port
+--
++
+Enter the following command to open the Cluster Network Operator (CNO) configuration in an editor:
```

Comment:
Users shall update MTU and Geneve port within the same patch command of changing the `NetworkType`. Users can choose to use either `oc patch` or `oc edit` with one transaction.  But not both of them, neither in separate transactions.

Reply:
@pliurh, I'm confused. I do not see any fields for MTU or Geneve in the `Network.config` CRD.

https://docs.openshift.com/container-platform/4.6/rest_api/config_apis/network-config-openshift-io-v1.html#network-config-openshift-io-v1

How to patch `Network.config` and `Network.operator` in the same command?

Thanks.

https://docs.openshift.com/container-platform/4.6/rest_api/operator_apis/network-operator-openshift-io-v1.html#network-operator-openshift-io-v1

Reply:
Oh, my fault. Please ignore that comment.

Reply:
But still, users shall execute these 2 commands together, so that the CNO can rendor the ovn-kube object correctly before applying it. So using 'oc patch' is more handy here, e.g. `oc patch Network.operator.openshift.io cluster --type='merge' --patch '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"mtu":1200}}}}'`

Reply:
@pliurh, I've made another update to this.

Reply:
There is a bug https://bugzilla.redhat.com/show_bug.cgi?id=1898159, which crashes the cluster if users do migrate with a new clusterNetwork CIDR and the new CIDR doesn't include the old CIDR. Shall we remove this session until that bug is closed?

Reply:
Sure, I've moved the offending text into a comment at the bottom of the module.